### PR TITLE
feature/loader

### PR DESCRIPTION
**Pull request by RomchikSt**
Trello: https://trello.com/c/RJoQac9I/18-implement-loading-animation

After:

https://github.com/unreal998/top-kolesa-fe/assets/140477189/0d717e95-44e3-46cb-9b87-63cac0627acb

### CODE REVIEW

File: src/modules/itemDetailsPage/index.tsx [hunk 1]
```diff
@@ -22,11 +22,13 @@ import { FullDescription } from "./components/FullDescription";
...
 import { useDispatch, useSelector } from "react-redux";
 import { actions } from "../shopPage/reducer";
-import { actions as itemDetailsActions } from "./reducer"
+import { actions as itemDetailsActions } from "./reducer";
 import { selectSelectedItemData } from "../shopPage/selectors";
 import { SHOP_ITEM_TIRES_IMG_PREFIX } from "../../constants";
 import { itemBuyDataBuilder } from "./utils/itemBuyDataBuilder";
 import { useNavigate } from "react-router";
+import Loader from "../../shared/components/Loader";
+/* import ClipLoader from "react-spinners/ClipLoader"; */
```
Comment (unreal998) [inline src/modules/itemDetailsPage/index.tsx:31]: Please , remove commented code

File: src/modules/itemDetailsPage/index.tsx [hunk 2]
```diff
@@ -52,229 +54,257 @@ export function ItemDetailsPage() {
...
-              borderStyle: "dashed",
-            }}
-          />
-          <Stack direction="row" justifyContent="space-between">
-            <Stack direction="row">
-              <Typography
-                fontSize="0.8rem"
-                color={BASE_COLORS.DEFAULT_GREY}
-                variant="body1"
+            <Stack
```
Comment (unreal998) [inline src/modules/itemDetailsPage/index.tsx:102]: Move that Stack , to the separate styled component

**Review by unreal998 — APPROVED** [review]
Fix my warnings and merge the you will be ready

File: src/modules/itemDetailsPage/index.tsx (hunk 1 above)
Comment (RomchikSt) [reply src/modules/itemDetailsPage/index.tsx:31]: done

**Pull request merged**